command/polly_tts.js

```diff
@@ -7,19 +7,23 @@ var glob = require('glob');
 var polly_tts = {
   getPollyChunk: function(text, filenameIndex, audio_file) {
     return new Promise(function(resolve, reject) {
+      let rate = process.env.PROSODY_RATE || 'medium';
+      var ssmlText =
+        '<speak><prosody rate="' + rate + '">' + text + '</prosody></speak>';
```

Comment:
Should we go ahead and add 'volume' while we're here?

Reply:
Yes, good call... I will add it.